src/ralph/data_importer/resources.py

```diff
@@ -233,7 +234,7 @@ class DataCenterAssetResource(RalphModelResource):
     rack = fields.Field(
         column_name='rack',
         attribute='rack',
-        widget=ImportedForeignKeyWidget(physical.Rack),
+        widget=RackWidget(physical.Rack),
```

Comment:
Does it work with CLI? with the flag MAP_IMPORTED_ID_TO_NEW_ID ? (import from old ralph)